website/docs/r/vulnerability_exception_host.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+subcategory: "Vulnerability Exceptions"
+layout: "lacework"
+page_title: "Lacework: lacework_vulnerability_exception_host"
+description: |- Create and manage Lacework Vulnerability Exception Hosts
+---
+
+# lacework\_vulnerability\_exception\_host
+
+Use this resource to create a Lacework Host Vulnerability Exception.
```

Comment:
Could we please elaborate more on this description?

Reply:
Added some additional description from the wip docs